tests/compile_ouroboros.py

```diff
@@ -0,0 +1,670 @@
+# -*- coding: utf-8 -*-
+"""Check modules from Ouroboros that compile with voc."""
+
+# Standard library imports
+from datetime import datetime
+from subprocess import Popen, PIPE
+import multiprocessing
+import os
+
+HERE = os.path.dirname(os.path.realpath(__file__))
+MODULES_WHITELIST = [
```

Comment:
Maybe it would be best to have the other way around: instead of a whitelist, this would be an "ignorelist" of the things we know we aren't compiling yet -- just so that it works also as a "TODO" list.